json-resume-builder/src/App.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styled from "styled-components";
+import resumeData from "./resume.json";
+
+import Header from "./components/Header";
+import Education from "./components/Education";
+import Summary from "./components/Summary";
+import Projects from "./components/Projects";
+import Work from "./components/Work";
+
```

Comment:
great use of componenets, I love seeing how you're utilizing react for this project it seems like it was extremely helpful!